web/ui/react-app/src/Navbar.tsx

```diff
@@ -14,7 +14,11 @@ import {
 } from 'reactstrap';
 import PathPrefixProps from './types/PathPrefixProps';
 
-const Navigation: FC<PathPrefixProps> = ({ pathPrefix }) => {
+interface NavbarConsolesLink {
```

Comment:
This is a duplicate of the one in ```App.tsx```

Reply:
Yeah, so what I meant was calling it `<Component>Props` if we don't make it a shared type, so it should be `NavbarProps` here, not `NavbarConsolesLink` (since it would over time also contain other props for the navbar, as necessary).

We could also make it a shared props type for it (similar to `PathPrefixProps`), but then it should live in one place and just be named `ConsolesLinkProps`.

Since it's only *really* needed by one component though, I'm kind of for just going for the `NavbarProps` variant that includes minor duplication...